### DISCUSSION TITLE
快捷键用不了

### DISCUSSION THREAD

**Discussion by ChaXxl** [the1812/Bilibili-Evolved#3378] [提问 / Q&A] [answered]
不知道为啥，点赞和投币的快捷键用不了，其它的都正常
版本：v2.1.10-preview
浏览器：百分浏览器 Chromium 86.0.4240.198
在EDGE也试了，也是用不了，不知道咋肥事，大伙支个招
![image](https://user-images.githubusercontent.com/40326898/171094998-79b72a75-13b9-46c4-84f7-00a6c3e2b7e2.png)

**Comment by Geeyun-JY3** [comment, marked as answer]
亲测确定是新版界面还没适配：#2615

**Reply by ChaXxl** [reply]
> 亲测确定是新版界面还没适配：#2615

哦，好吧。现在白嫖有理有据了----我投币的快捷键用不了
![image](https://user-images.githubusercontent.com/40326898/171130510-f9146a0c-4687-4187-98ec-641a2416e894.png)

**Reply by Geeyun-JY3** [reply]
非要用快捷键的话官方是有三连的快捷键的：#2231（doge
